as_perimenoun_oi_gynaikes.ass

```diff
@@ -278,14 +278,14 @@ Dialogue: 0,0:17:53.78,0:17:57.66,Default,,0,0,0,,{\i1}Now, I pay for my mistake
 Dialogue: 0,0:17:57.66,0:18:00.30,Default,,0,0,0,,{\i1}I regret it all. I regret it all.{\i0} \NDon't pick up!
 Dialogue: 0,0:18:04.42,0:18:11.18,Default,,0,0,0,,Antonis? Panos had a sudden malaise. He was feeling down and \Nhis blood sugar got low. We will be in Kavala...
 Dialogue: 0,0:18:11.20,0:18:14.82,Default,,0,0,0,,...in an hour, with the ferry we will be at Thasos at around three or four.
-Dialogue: 0,0:18:14.82,0:18:18.68,Default,,0,0,0,,What the hell are you telling him? I could be sitting here with a blood-pressure in the low twenties.
+Dialogue: 0,0:18:14.82,0:18:18.68,Default,,0,0,0,,What the hell are you telling him? I could be sitting here with a blood-pressure above twenties!

```

Comment:
This is syntactically wrong and does not improve the original.

Reply:
Please elaborate....
High blood-pressure (>20) is a problem and that's what the dialog says.
What exactly "in the low twenties" means?

Reply:
It means a number in the low range of 20-29.

https://forum.wordreference.com/threads/is-in-the-low-20s.3531907/